docs/pages/repo/docs/handbook/environment-variables.mdx

```diff
@@ -28,6 +28,23 @@ We're actively looking forward to improving the developer experience and ergonom
 }
 ```
 
+## Accessing root .env from sub-packages
```

Comment:
```suggestion
## Accessing root .env from other workspaces
```

Reply:
How about "Accessing a root `.env` in a workspace script"? Trying to avoid confusion about what this header would mean.